languages/elixir/exercises/concept/tuples/.meta/design.md

```diff
@@ -0,0 +1,47 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of tuples and common conventions in elixir to return multiple values.
+
+This is done through the implementation of a basic kitchen calculator.
+
+## Learning objectives
+
+- Know what a tuple is
+- Know how to write a tuple literal
+- Know how to return a tuple from a function
+- Know that returning a tuple is a common for of multiple return in Elixir
+- Know how to extract an element from a tuple using `elem/2`
+- Know how to write a floating point literal
+- Know how to round a floating point number to decimal place or integer
+
+## Out of scope
+
+- case do pattern matching
+
+## Prerequisites
+
+The exercise's prerequisites are:
+
+- `basics`: needs to be able to define named functions, define and return integers, return values from functions
```

Comment:
I had the impression that `basics` is a special requirement that only needs to be listed if there are no other requirements. Otherwise, we might need to add it to every single exercise.

Reply:
I don't think I have that listed in the config, so I think this was an oversight that I forgot to remove.  You are correct, you should only list direct parents in the graph.

Reply:
I think it's perfectly fine to have it listed here, but to omit it from the `config.json` (which I think you've done). Whatever works for you.